src/output/dnssim.c

```diff
@@ -136,6 +138,12 @@ void output_dnssim_free(output_dnssim_t* self)
     free(self);
 }
 
+void output_dnssim_log_name(output_dnssim_t* self, const char* name)
+{
+    snprintf(self->_log.name, LOG_NAME_MAXLEN, "%s", name);
```

Comment:
Why `snprintf()`? A `memcpy()` will work.

And remove `LOG_NAME_MAXLEN` (from `log.h` also), use `sizeof(self->_log.name)` instead.

Reply:
fixed